code/modules/mod/mod_construction.dm

```diff
@@ -6,18 +6,34 @@
 	name = "MOD helmet"
 	icon_state = "helmet"
 
+/obj/item/mod/construction/helmet/examine(mob/user)
+	. = ..()
+	. += span_notice("You could insert it into a <b>MOD shell</b>...")
```

Comment:
Is there some reason it's not on the base path ?

Reply:
the shell itself is a construction item, and it wouldnt fit with plural things like boots or gauntlets